src/axom/spin/BVH.hpp

```diff
@@ -140,6 +140,7 @@ class BVH
    *
    * \param [in] boxes buffer consisting of bounding boxes for each entity.
    * \param [in] numItems the total number of items to store in the BVH.
+   * \param [in] allocatorID specify the ID of the allocator to use. Optional.
```

Comment:
Similar comment to above: Umpire ID, correct?

Reply:
Fixed.